test/powershell/Modules/Microsoft.PowerShell.Management/Split-Path.Tests.ps1

```diff
@@ -69,4 +69,53 @@ Describe "Split-Path" -Tags "CI" {
 	Split-Path -Parent "/usr/local/bin"  | Should Be "${dirSep}usr${dirSep}local"
 	Split-Path -Parent "usr/local/bin"   | Should Be "usr${dirSep}local"
     }
+
+    Context "SplitPath Tests (Windows Only)" {
```

Comment:
Why it's Windows Only? Can we cover all systems in these tests? I think the attitude of https://github.com/PowerShell/PowerShell/issues/2301#issuecomment-261118596 is to provide a consistent experience on all platforms.